DecipherNow/gm-fabric-dashboard
# Graph areas filled by default

Resolves #1451

## Proposed Changes
  - Declare graph areas 'filled' by default
  
## How to Test

  - View any graph with varying data
  - Observe area under graphed line should be filled in color.
<img width="1454" alt="screen shot 2018-02-01 at 9 24 39 am" src="https://user-images.githubusercontent.com/98312/35683396-c0a286f8-0731-11e8-8352-cd30a561fa06.png">
